utils.js

```diff
@@ -375,7 +375,8 @@ function formatDeltaMessage(m){
     messageID: md.messageId,
     attachments: (m.delta.attachments || []).map(v => _formatAttachment(v)),
     timestamp: md.timestamp,
-    isGroup: !!md.threadKey.threadFbId
+    isGroup: !!md.threadKey.threadFbId,
+    reactions: originalMessage.reactions ? originalMessage.reactions : []
```

Comment:
Sorry, I don't understand. Can you explain more?  Does delta message not have info about reactions?
Mhh... because it sends the delta message right away, so no one could have reacted to it yet?